pkg/apis/v1beta1/suite_test.go

```diff
@@ -21,12 +21,21 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	. "knative.dev/pkg/logging/testing"
+
+	"github.com/aws/karpenter-core/pkg/apis"
+	"github.com/aws/karpenter-core/pkg/operator/scheme"
+	"github.com/aws/karpenter-core/pkg/test"
 )
 
 var ctx context.Context
+var env *test.Environment
 
 func TestAPIs(t *testing.T) {
 	ctx = TestContextWithLogger(t)
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "v1beta1")
 }
+
+var _ = BeforeSuite(func() {
```

Comment:
I believe there is an `env.Stop()` call that we need to make sure that we call at the end. We should probably also call `ExpectCleanedUp` in the `AfterEach` just by convention